test/incrParse/Outputs/extend-identifier-at-eof.json

```diff
@@ -1,33 +1,33 @@
 {
-  "id": 39,
```

Comment:
Are these id changes due to we're no longer discarding node?

Reply:
Correct.